src/meshapi/management/commands/scramble_members.py

```diff
@@ -37,7 +36,9 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             help="Skip scrambling installs",
         )
 
-    @transaction.atomic
+    # This transaction is not atomic so that even if there's some error with
+    # the scrambled data, we can scramble it partially. I'd rather break someone's
+    # dev env than give someone PII accidentally
```

Comment:
Doesn't removing the atomic block make it more likely that we will disclose PII? Since a single row could fail silently?

Reply:
whoops, I forgot to push again. I have another commit that sets up atomic for each individual transaction.